ansible/configs/ocp-ha-lab/files/hosts_template.3.9.14.j2

```diff
@@ -210,16 +159,13 @@ openshift_logging_curator_nodeselector={"env":"infra"}
 # Configure additional projects
 openshift_additional_projects={'openshift-template-service-broker': {'default_node_selector': ''}}
 
-
 ###########################################################################
 ### OpenShift Router and Registry Vars
 ###########################################################################
 
-openshift_hosted_router_selector='env=infra'
```

Comment:
Do we want to remove this line ?

Reply:
let's keep it for now, we can remove it when we come up with a better line. 
Eric S suggested we change it to "type" rather than "env"